src/Analyzers/CSharp/Tests/UseCollectionInitializer/UseCollectionInitializerTests.cs

```diff
@@ -78,6 +84,76 @@ void M()
                 """);
         }
 
+        [Fact]
+        public async Task TestOnVariableDeclarator_AddRange()
+        {
+            await TestInRegularAndScriptAsync(
+                """
+                using System.Collections.Generic;
+
+                class C
+                {
+                    void M(int[] x)
+                    {
+                        var c = [|new|] List<int>();
+                        [|c.Add(|]1);
+                        c.AddRange(x);
```

Comment:
these tests validate that the stuff that collection expressions *would* use, are *not* used when doing collection *initializers*.